lms/product/canvas/_plugin/grouping.py

```diff
@@ -78,8 +79,16 @@ def get_groups_for_instructor(self, _svc, course, group_set_id):
                 self._custom_course_id(course), group_set_id
             )
         except CanvasAPIError as canvas_api_error:
+            group_set_name = None
+            if group_set := self._request.find_service(name="course").find_group_set(
+                group_set_id=group_set_id
+            ):
+                group_set_name = group_set["name"]
+
             raise GroupError(
-                ErrorCodes.GROUP_SET_NOT_FOUND, group_set=group_set_id
+                ErrorCodes.GROUP_SET_NOT_FOUND,
+                group_set=group_set_id,
+                group_set_name=group_set_name,
```

Comment:
The name we are looking for is pretty important, more so in Canvas which doesn't copy the group sets so they need to be created by hand.

Expose the old name if we have it.